lamindb/curators/__init__.py

```diff
@@ -102,10 +101,10 @@ class CurateLookup:
         slots: A dictionary of slot fields to lookup.
         public: Whether to lookup from the public instance. Defaults to False.
 
-    Example:
-        >>> curator = ln.Curator.from_df(...)
-        >>> curator.lookup()["cell_type"].alveolar_type_1_fibroblast_cell
-        <Category: alveolar_type_1_fibroblast_cell>
+    Example::
+
+        curator = ln.Curator.from_df(...)
```

Comment:
Should replace with `curator = ln.curators.DataFrameCurator(...); curator.cat.lookup()`